srcpkgs/ulogd/template

```diff
@@ -1,9 +1,10 @@
 # Template file for 'ulogd'
 pkgname=ulogd
 version=2.0.7
-revision=2
+revision=3
 build_style=gnu-configure
 configure_args="--sbindir=/usr/bin"
+conf_files="/etc/ulogd.conf"
```

Comment:
Shouldn't the logrotate file also be a conf one?

Reply:
Added the logrotate file as well. Thanks.